whist/server/database/user.py

```diff
@@ -0,0 +1,28 @@
+"""User models"""
+from typing import Optional
+
+from pydantic import BaseModel, Field
+
+from whist.server.database.id_wrapper import PyObjectId
+from whist.server.services.password import PasswordService
+
+
+class User(BaseModel):
+    """User DAO"""
+    id: Optional[PyObjectId] = Field(alias='_id')
+    username: str
+
+
+class UserInDb(User):
+    """
+    User DO
+    """
+    hashed_password: str
+
+    def verify_password(self, password):
```

Comment:
A test for this is moved to https://github.com/Whist-Team/Whist-Server/issues/65